packages/runtime/src/schema.ts

```diff
@@ -178,6 +178,27 @@ const UpdateOptionsInput = new GraphQLInputObjectType({
   },
 })
 
+const CeramicSignerInputType = new GraphQLInputObjectType({
+  name: 'CeramicSignerInput',
+  fields: {
+    isAuthenticated: { type: GraphQLBoolean },
+  },
+})
+
+const HideOptionsInput = new GraphQLInputObjectType({
+  name: 'HideOptionsInput',
+  fields: {
+    asDID: {
+      type: GraphQLDID,
+      description: 'DID for the signer(Optional)',
+    },
+    signer: {
+      type: CeramicSignerInputType,
+      description: 'Ceramic signer (Optional)',
+    },
```

Comment:
Same as previous comment, this seems to be information we get from the context, not an input